doc/source/management/data_at_rest_encryption.rst

```diff
@@ -477,202 +597,118 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. rubric:: Redo log encryption
 
+:Availability: This feature is **Experimental** quality.
 
-Temporary file encryption
-================================================================================
-
-:Availability: This feature is of **Beta** quality.
-
-The encryption of temporary files is triggered by the
-:variable:`encrypt-tmp-files` option.
-
-Temporary files are currently used in |Percona Server| for the following
-purposes:
-
-* filesort (for example, ``SELECT`` statements with ``SQL_BIG_RESULT`` hints),
-* binary log transactional caches,
-* Group Replication caches.
+Implemented in :rn:`8.0.16-7`, the key rotation for the redo log has been
+redesigned to rotate the key with `SELECT rotate_system_key("percona_redo").
```

Comment:
rotate_system_key is still mentioned here. @robgolebiowski should we remove this paragraph too? Or just mark it as experimental?